slides/testing/parser/file-parser.ts

```diff
@@ -0,0 +1,16 @@
+import { File } from './file';
+import { IParser } from './parser.interface';
+
+export class FileParser implements IParser<Array<File>> {
+
+  parse(data: any): Array<File> {
+    let slides = data.content.match(/<!-- .slide: id.*-->/g) || [];
```

Comment:
I know right now we have separate comments for slide background and slide id like...

```html
<!-- .slide: data-background="../content/images/title-slide.jpg" -->
<!-- .slide: id="redux" -->
```
but this would also be equally valid
`<!-- .slide: id="redux" data-background="../content/images/title-slide.jpg" -->`
and
`<!-- .slide: data-background="../content/images/title-slide.jpg" id="redux" -->`

It may be worth making this regex more intelligent in that it really looks for _just_ the id between `id=""`